frontend/src/api/apiClient.ts

```diff
@@ -1,13 +1,12 @@
 import axios from 'axios';
 import requestInterceptor from './middleware/requestInterceptor';
 import errorInterceptor from './middleware/errorInterceptor';
-import forbiddenErrorInterceptor from './middleware/forbiddenErrorInterceptor';
 
 const apiClient = axios.create({
-  baseURL: process.env.BASE_URL || 'http://localhost:8000',
+  baseURL: 'http://localhost:8000',
```

Comment:
Удалять эту вещь не стоит, она пригодится при написании конечного docker compose. Однако стоит поменять следующим образом (см. https://vite.dev/guide/env-and-mode) (ну или иным образом)
```suggestion
  baseURL: import.meta.env.VITE_BASE_URL || 'http://localhost:8000',
```